package.json

```
@@ -7,10 +7,18 @@
     "dev": "./node_modules/.bin/rollup -c -w",
     "build": "npm run prettier:fix && ./node_modules/.bin/rollup -c",
     "test": "jest",
```

I think we can add `test:watch` script too

```suggestion
    "test": "jest",
    "test:watch":"npm t -- --watch",
```

---

I had missed that. Thank you!